views/term_table.erb

```diff
@@ -246,3 +246,5 @@
             here's <a href="http://docs.everypolitician.org/contribute.html">how to get that fixed</a>.</p>
         </div>
       </div>
+      
+
```

Comment:
Trailing whitespace klaxon! 🚨 

Probably best to remove this so the diff only contains the changes you've made.